sites/friday/src/get-started/strapi/_index.md

```diff
@@ -1,4 +1,591 @@
+ > 0.1
+```
+
+Next it will ask how many instances of our application container we need deployed. For now let's go with `1`:
+```bash {location="Terminal"}
+Enter the number of instances (default: 1): 1
+```
+
+And finally, it will ask how many disk space of our application container we need deployed. For now let's go with `512`:
+```bash {location="Terminal"}
+Enter a disk size in MB : 512
+```
+
+Let's allocate resources to our ``postgresql`` service.
+The first question is what profile size you want applied to your service image. For now let's select the minimum `0.1`:
+
+```bash {location="Terminal"}
+Service: postgresql
+
+Choose a profile size:
+[0.1 ] CPU 0.1, memory 448 MB
+[0.25] CPU 0.25, memory 832 MB
+[0.5 ] CPU 0.5, memory 1408 MB
+[1   ] CPU 1, memory 2432 MB
+[2   ] CPU 2, memory 4032 MB
+[4   ] CPU 4, memory 6720 MB
+[6   ] CPU 6, memory 9024 MB
+[8   ] CPU 8, memory 11200 MB
+[10  ] CPU 10, memory 13184 MB
+> 1
+```
+And finally, it will ask how many disk space of our service container we need deployed. For now let's go with `512`:
+```bash {location="Terminal"}
+Enter a disk size in MB: 512
+```
+
+Last it will ask us to confirm our choices. Select `Y` and the {{% vendor/name %}} will take your selections, grab the
+previous built images from early, apply our resource selections to them and deploy our full application!
+```javascript
+Summary of changes:
+  App: app
+    CPU: 0.1
+    Memory: 64 MB
+  Service: postgresql
+    CPU: 0.1
+    Memory: 448 MB
+    Disk: 512 MB
+
+Are you sure you want to continue? [Y/n] y
+```
+
+Then it will automatically redeploy your ``main`` environment with allocated resources.
+
+Note that each environment has its own domain name.
+To open the url of your new environment, run the following command:
+
+   ```bash {location="Terminal"}
+   {{% vendor/cli %}} environment:url --primary
+   ```
+
+Et voilà, your Strapi application is live!
+
+## TODO Make changes to your project
+
+--------
+Now that your project is deployed, you can start making changes to it.
+For example, you might want to fix a bug or add a new feature.
+
+In your project, the `main` branch always represents the production environment.
+Other branches are for developing new features, fixing bugs, or updating the infrastructure.
+
+To make changes to your project, follow these steps:
+
+1. Create a new environment (a Git branch) to make changes without impacting production:
+
+   ```bash {location="Terminal"}
+   {{% vendor/cli %}} branch feat-a
+   ```
+
+   This command creates a new local `feat-a` Git branch based on the main Git branch
+   and activates a related environment on {{< vendor/name >}}.
+   The new environment inherits the data (service data and assets) of its parent environment (the production environment here).
+
+2. Make changes to your project.
+
+   For example, edit the `./index.js` file and make the following changes:
+
+   ```javascript {location="index.js"}
+   app.get('/', async function(req, res){
```

Comment:
I'm assuming this is where you ended since the strapi app doesnt have an `index.js` file in the project root?

Reply:
Yep, that's why i add a big TODO in the title of the `Make change to your project`.

Reply:
modification in Strapi source code is a lot more complex so i just add a link to the Strapi official doc to customize the admin panel.